test/EndToEnd/ProjectTemplates/UwpPackageRefClassLibrary.zip/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
```

Comment:
but that destroys the integrity of the template right? other people might write tests which rely on the existence of this file.